src/theme/components/multiSelectItemsCountButton.ts

```diff
@@ -13,7 +13,7 @@ const MultiSelectItemsCountButton = defineStyleConfig({
     fontSize: "desktop.caption",
     justifyContent: "flex-end",
     left: "15px",
-    marginRight: "xs",
+    marginRight: "m",
```

Comment:
Any thoughts on what would have caused this shift in the spacing?

Reply:
Perhaps a merge conflict?

Reply:
I didn't see this issue in Alkim's notes. Did I miss it or is this from somewhere else?

Reply:
Ah I see now. Thank you!